docs/oscar_references.bib

```diff
@@ -448,6 +448,16 @@ @InCollection{Chr91
   doi           = {10.1007/BFb0086376}
 }
 
+@Article{CMSS07,
+	author = {Collins, Benoît and Mingo, James A. and Śniady, Piotr and Speicher, Roland},
+	title = {Second order freeness and fluctuations of random matrices. {III}: {Higher} order freeness and free cumulants},
```

Comment:
```suggestion
	title = {Second order freeness and fluctuations of random matrices. III: Higher order freeness and free cumulants},
```
The bibtex Parser does not like braces, but instead keeps capitalization by itself.

Reply:
And there are more bibtex issues (see the bibtool test ci job for details)

Reply:
see also https://docs.oscar-system.org/dev/DeveloperDocumentation/documentation/#Updating-the-bibliography